ush/job_preamble.sh

```diff
@@ -120,6 +120,8 @@ else
     export COMOUT="${COMOUT_BASEDIR}/${PDY}${cyc}"
     export COMINext="${EXTROOT}/${PDY}${cyc}"
 fi
+export COMIN_PDY="${COMIN_BASEDIR}/${RUN}.${PDY}"
+export COMIN_PDYm1="${COMIN_BASEDIR}/${RUN}.${PDYm1}"
```

Comment:
I see what you mean by PDYm1 not being defined without "setpdy.sh" called. I don't think it is unreasonable to set PDYm1 using dateutil since `setpdy.sh` is not available on all platoforms and also require some changes to it to.

Reply:
I've added PDYmX for the community mode.